jax/interpreters/sharded_jit.py

```diff
@@ -249,11 +249,20 @@ def sharded_jit(fun: Callable, in_parts, out_parts):
     out_parts: The output partitions, i.e. how each output of ``fun`` should be
       partitioned or replicated. This follows the same convention as
      ``in_parts``.
+    num_partitions: Optional. If set, explicitly specifies the number of devices
+      ``fun`` should partitioned across (rather than inferring it from
```

Comment:
```suggestion
      ``fun`` should be partitioned across (rather than inferring it from
```

Reply:
Oops I had an old tab opened that showed you approved but didn't show this comment :( I'll fix it next time I'm in the area.